lib/block_view.go

```diff
@@ -543,6 +544,16 @@ func (pe *PostEntry) IsDeleted() bool {
 	return pe.isDeleted
 }
 
+func (pe *PostEntry) HasMedia() bool {
+	bodyJSONObj := BitCloutBodySchema{}
+	err := json.Unmarshal(pe.Body, &bodyJSONObj)
+	//Only return true if json can be parsed and ImageUrls is not nil/non-empty
+	if err != nil || bodyJSONObj.ImageURLs == nil || len(bodyJSONObj.ImageURLs) == 0 {
```

Comment:
do you want posts with embedded youtube, vimeo, tiktok, etc links? if so, you'll need to check the ExtraData for `EmbedVideoURL`

Reply:
Ah yea we do, thanks for the call out. Will add.